<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorForm</class>
 <widget class="QWidget" name="EditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File editor</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>571</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Edit your file, pozhaluista</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Times</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Helvetica [Cronyx]</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveAsButton">
        <property name="text">
         <string>Save as</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeEditorButton">
        <property name="text">
         <string>Close editor</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
